modules/renter/downloadcache_test.go

```diff
@@ -0,0 +1,39 @@
+package renter
+
+import (
+	"strconv"
+	"sync"
+	"testing"
+	"time"
+)
+
+// TestAddChunkToCache tests that the oldest chunk is removed
+func TestAddChunkToCache(t *testing.T) {
+	udc := &unfinishedDownloadChunk{
+		download: &download{
+			staticDestinationType: "httpseekstream",
```

Comment:
This should use the `destinationTypeSeekStream` constant instead of a hardcoded value.